.gitignore

```diff
@@ -0,0 +1,108 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
let's make the `.gitignore` minimalist. So probably only ignore `__pycache__/` for now

Reply:
> 
> 
> Thanks for the PR @wagenrace - a couple of important comments below:
> 
>     1. This code is used to **download** the data on figshare, not **process** the data to upload to figshare. In #4, I am specifically interested in adding the code that @fheigwer used to process the data that was presumably downloaded from IDR (DOI: http://doi.org/10.17867/10000101). I think changing the naming conventions would make this much clearer. Perhaps instead of `0.process-data` it should be `0.process-idr-images`. To merge this PR, please change the module to `1.download-data`.
> 
>     2. I am thinking that the function `downloadData()` in `main.py` will never be used outside the `1.download-data` module - lets put this file in a folder called `1.download-data/scripts`
> 
>     3. I was probably not being 100% clear, but in an analysis module repository style, each folder _stands alone_. Therefore, there should not be any need to import functions across folders and the current file `0process-data/__init__.py` is not necessary.
> 
>     5. Please add execution instructions. This could be in the form of a jupyter notebook that, when run, will download.

I think I better understand you. I will migrate this code to 1.download-data. And document is more clearly.

Reply:
I do believe in keeping things minimal but I belief the .gitignore is the exception to this rule. I do not want my vscode file to end up in here or other data files. I will try to minimize is a bit futher but will always be longer as  "__pycache__/"